draft-ietf-quic-manageability.md

```diff
@@ -167,13 +165,15 @@ The following information is exposed in QUIC packet headers:
   Header Form and fixed bits. Header types correspond to stages of the
   handshake; see Section 17.2 of {{QUIC-TRANSPORT}} for details.
 
-- version number: the version number present in the long header, and identifies
+- version number: the version number is present in the long header, and identifies
   the version used for that packet. Note that during Version Negotiation (see
-  {{version}}, and Section 17.2.1 of {{QUIC-TRANSPORT}}, the version number
+  {{version}} and Section 17.2.1 of {{QUIC-TRANSPORT}}), the version number
   field has a special value (0x00000000) that identifies the packet as a Version
   Negotiation packet. QUIC versions that start with 0xff are IETF drafts. QUIC
   versions that start with 0x0000 are reserved for IETF consensus documents,
-  for example the QUIC version 1 is expected to use version 0x00000001.
+  for example the QUIC version 1 is expected to use version 0x00000001. Operators
```

Comment:
```suggestion
  For example the QUIC version 1 uses version 0x00000001. Operators
```

If you are removing "QUIC is a moving target", then this will be the case.

Unless it isn't, but that will be a minor catastrophe that we can work through if it happens.